application/event_processor/event_processor.py

```diff
@@ -104,14 +105,14 @@ def send_changes(self) -> None:
 
 
 @tracer.capture_lambda_handler()
-@logger.inject_lambda_context(correlation_id_path="correlation_id")
```

Comment:
please leave in 
`@logger.inject_lambda_context` decorator without the correlation id this ensures many other useful context information is added to the log recrord such as functionName request id etc.